src/ucp/core/ucp_ep.h

```diff
@@ -411,9 +412,7 @@ struct ucp_ep_config {
  */
 typedef struct ucp_ep {
     ucp_worker_h                  worker;        /* Worker this endpoint belongs to */
-
-    uint8_t                       refcount;      /* Reference counter: 0 - it is
-                                                    allowed to destroy EP */
+    /* Place for 1-byte field is vacant here */
```

Comment:
are you sure? Looks like in release mode we had one extra byte here, so can make refcnt `uint16_t` and leave it here

Reply:
yes, you're right. we could move it here.
but what if we post more than 65535 send operations the same buffer?
it will break it again, since we will do validation for all requests in case of error.
also, better to keep 16-bits available for more critical fields

Reply:
but using size_t also increase overall ep footprint. 
Can we think on some way to not use ep refcounts for invalidations?

Reply:
I found the way how to decrease overall EP size - see the latest commit

Reply:
> Can we think on some way to not use ep refcounts for invalidations?

I bit weird solution - https://github.com/openucx/ucx/pull/7714
I guess `refcount` could be used in many other places in the future, so having it `size_t` will help us

Reply:
i like #7714, imo it is better, also removing one of refcount entities